docs/partial_source/contributing/setting_up.rst

```diff
@@ -34,6 +34,27 @@ Depending on your preferred mode of cloning, any of the below should work:
 
     gh repo clone --recurse-submodules unifyai/ivy your_folder
 
+A Free Guide To Setting Up
```

Comment:
I think the header should be "Setting up for Free" and we should mention something along the lines of "Pycharm professional license is only free for students and VSCode is a free alternative". Also the positioning of this section should be below PyCharm given that PyCharm should still be the one that's recommended.

Reply:
Done!
Do let me know if any other changes are required. Thanks 🙂